### PR TITLE
[LM-9392] Data Wrapper Model and UUID changes

### PR DESCRIPTION
**Pull request by arnavr15**
## Description

**Features**
- Model changes according to the data wrapper class
- UUID changes

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: ChatSX/store/reducers/fileUploadReducer.ts [hunk 1]
```diff
@@ -49,13 +49,13 @@ export function fileUploadReducer(state = initialState, action: any) {
...
     case CLEAR_FILE_UPLOADING_MESSAGES: {
       const {ID} = action.body;
-
       let obj = removeKey(ID, state.uploadingFilesMessages);
       let dummyState = {
         ...state,
         uploadingFilesMessages: {...obj},
       };
 
+      // ??
```
Comment (jaixchaudhary) [inline ChatSX/store/reducers/fileUploadReducer.ts:58]: type the use case of the function

File: index.js [hunk 2]
```diff
@@ -28,7 +28,9 @@ messaging().setBackgroundMessageHandler(async remoteMessage => {
   return val;
 });
 
-export const myClient = LMChatClient.setApiKey('')
+export const myClient = LMChatClient.setApiKey(
```
Comment (jaixchaudhary) [inline index.js:31]: remove api key

File: index.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
 /**
  * @format
  */
-
 import {AppRegistry} from 'react-native';
 import App from './App';
 import {name as appName} from './app.json';
 import messaging from '@react-native-firebase/messaging';
-import LMChatClient from '@likeminds.community/chat-js';
+// import LMChatClient from '@likeminds.community/chat-js';
```
Comment (jaixchaudhary) [inline index.js:8]: remove unnecessary comments

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: ChatSX/components/Poll/CreatePollScreen/index.tsx [hunk 4]
```diff
@@ -291,6 +293,12 @@ const CreatePollScreen = ({navigation, route}: CreatePoll) => {
         expiryTime: Date.parse(time.toString()),
       };
       const res = await myClient.postPollConversation(payload);
+      let payload1 = {
```
[the remark this answers is not in the document]
Comment (arnavr15) [reply ChatSX/components/Poll/CreatePollScreen/index.tsx]: changed to `getConversationPayload`

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**